.travis.yml

```diff
@@ -22,6 +22,16 @@ jobs:
       cache: pip
       install: pip install moban
       script: .ci/check_moban.sh
+    - stage: coala
+      <<: *disable_global
+      language: python
+      python: 3.6
+      cache: pip
+      install:
+        - BEARS_ZIP_URL=https://codeload.github.com/coala/coala-bears/zip
+        - curl -fsSL -o coala-bears.zip $BEARS_ZIP_URL/master
+        - pip install coala-bears.zip[alldeps] --pre -U
```

Comment:
this installs `coala` from pypi, which is incompatible.

Please also install `coala` from master using the same sequence.

This repo is blocked until you fix this.

Reply:
I am slightly confused. https://codeload.github.com/coala/coala-bears/zip returns 400: Invalid request. By a similar sequence, do you want me to include a sequence for `coala` like:

`- COALA_ZIP_URL=https://codeload.github.com/coala/coala/zip`
`- curl -fsSL -o coala.zip $COALA_ZIP_URL/master`
`- pip install coala.zip[alldeps] --pre -U`

or 

`- python setup.py bdist_wheel`
`- pip install ./dist/coala-*.whl`

?
 Sorry for the delay. Thanks

Reply:
I am not sure where `codeload.github.com` comes from.  The objective is to install coala master , using zips.  The zip links can be found on https://github.com/coala/coala/ .  Once you have the .zip file on the CI machine, installing it is a matter of using whatever pip & setuptools supports to install it.

Reply:
I think you need to specify `master` in the link, as implemented in the current scenario. Installing coala before coala-bears in the same way works fine for me, also fixes the CI.

Reply:
Hey @abhishalya I have added statements to download coala-master using zip and install it with pip. Would u mind taking a look and tell me what u did so the CI was successful ? Thanks